### DISCUSSION TITLE
Нумерация документов в пределах года.

### DISCUSSION THREAD

**Discussion by pitermax1977** [totumonline/totum-ru-issues-and-discussions#527] [Как это сделать?] [answered]
Здравствуйте! Возможно ли сделать нумерацию документов в пределах года.
То есть при создании документа присваивался ему уникальный номер, а
со следующего года нумерация начиналась опять с единицы.
Надеюсь объяснил понятно

**Comment by erpic-ru** [comment, marked as answer]
https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/103

**Comment by pitermax1977** [comment]
Спасибо! Буду разбираться
